savefiledecoder/PathHelper.cs

```diff
@@ -14,6 +14,8 @@ public static class PathHelper
         public const string ExportObjectivesFileName = "export_objectives.txt";
         public const string ExportCheckpointsFileName = "export_checkpoints.txt";
         public const string ExportVariablesFileName = "export_variables.txt";
+        public const string ExportFlagsFileName = "export_flags.txt";
+        public const string ExportFloatsFileName = "export_floats.txt";
 
```

Comment:
It could be a good idea to export data to a separate "Export" folder. Then we can remove "export_" prefix from file names.
Just an idea, no pressure.

Reply:
I think it's a good idea, also makes the files more easily visible compared to when they are among all those DLLs.